infores_catalog.yaml

```diff
@@ -2927,7 +2927,7 @@ information_resources:
     agent type: not_provided
     description: SPOKE KP - an NIH NCATS Knowledge Provider to expose UCSFs SPOKE
     xref:
-      - https://spoke.ucsf.edu/
+      - https://github.com/NCATSTranslator/Translator-All/wiki/imProving-Agent
```

Comment:
Based on our Slack exchange with Brett, I think the correct URL is https://github.com/NCATSTranslator/Translator-All/wiki/SPOKE-KP.